_resources/port1.0/group/qt6_version_info-1.0.tcl

```diff
@@ -5,7 +5,7 @@
 
 global available_qt_versions
 array set available_qt_versions {
-    qt6 {qt6-qtbase 6.2}
+    qt6 {qt6-qtbase 6.5}
```

Comment:
Why can't we make Qt 6.5 a separate version like in the [qt5_version_info](https://github.com/macports/macports-ports/blob/1719a40fce26c09ed29c4ecda3fca6b2fd1bc7a7/_resources/port1.0/group/qt5_version_info-1.0.tcl) portgroup?

Reply:
That is the plan.  This was just a placeholder for me to keep moving forward while that path forward gets sorted.

Here's the thread on the [dev list](https://lists.macports.org/pipermail/macports-dev/2023-April/044984.html)

Reply:
I've just pushed the logic to allow macOS version < 11 to build 6.2 instead